src/main/java/marquez/api/resources/JobResource.java

```diff
@@ -213,8 +213,8 @@ public Response failJobRun(@PathParam("runId") final String runId) throws Resour
 
   @PUT
   @Timed
-  @Path("/jobs/runs/{runId}/abort")
-  public Response abortJobRun(@PathParam("runId") final String runId) throws ResourceException {
+  @Path("/jobs/runs/{id}/abort")
+  public Response abortJobRun(@PathParam("id") final String runId) throws ResourceException {
```

Comment:
This will break integrations, but it's not a hard change to adjust to,.

Reply:
Which integration test would break?

Reply:
Sorry, on second thought, it wouldn't break any tests b/c it's a pathparam, defined right above. This change is fine.